Adapt downsteam Ghost Squid support to latest QMK (#14607)
qmk/qmk_firmware: keyboards/bpiphany/ghost_squid/readme.md

# Ghost Squid Controller

This is the firmware for Rev. 20140518 of the Ghost Squid controller by [Bathroom Epiphanies](http://bathroomepiphanies.com/controllers/), a replacement controller for the [Cooler Master Quick Fire XT](https://www.coolermaster.com/catalog/peripheral/keyboards/quick-fire-xt/).

The code was adapted from the [BathroomEpiphanies QMK Firmware](https://github.com/BathroomEpiphanies/epiphanies_qmk_keyboard/tree/master/keyboards/ghost_squid_20140518).

* Keyboard Maintainer: QMK Community
* Hardware Supported: Ghost Squid
* Hardware Availability: [1upkeyboards](https://1upkeyboards.com/shop/controllers/qf-xt-ghost-squid-controller-2/)

> Make example for this keyboard (after setting up your build environment):

Reviewer: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Author: Thanks, added.

# Ghost Squid Controller

> This is the firmware for Rev. 20140518 of the Ghost Squid controller
> by [Bathroom Epiphanies](http://bathroomepiphanies.com/controllers/),
> a replacement controller for the [Cooler Master Quick Fire
> XT](https://www.coolermaster.com/catalog/peripheral/keyboards/quick-fire-xt/).
> 
> The code was adapted from the [BathroomEpiphanies QMK
> Firmware](https://github.com/BathroomEpiphanies/epiphanies_qmk_keyboard/tree/master/keyboards/ghost_squid_20140518).

Reviewer 2: ```suggestion
This is the firmware for Rev. 20140518 of the Ghost Squid controller by [Bathroom Epiphanies](http://bathroomepiphanies.com/controllers/), a replacement controller for the [Cooler Master Quick Fire XT](https://www.coolermaster.com/catalog/peripheral/keyboards/quick-fire-xt/).

The code was adapted from the [BathroomEpiphanies QMK Firmware](https://github.com/BathroomEpiphanies/epiphanies_qmk_keyboard/tree/master/keyboards/ghost_squid_20140518).
```

> ## Layout support
> 
> Support for both 104 key layout is provided. See the
> keymaps `default` for example layout.
> 
> Keyboard Maintainer: QMK Community
> Hardware Supported: Ghost Squid
> Hardware Availability: [1upkeyboards](https://1upkeyboards.com/shop/controllers/qf-xt-ghost-squid-controller-2/)

Reviewer 2: ```suggestion
* Keyboard Maintainer: QMK Community
* Hardware Supported: Ghost Squid
* Hardware Availability: [1upkeyboards](https://1upkeyboards.com/shop/controllers/qf-xt-ghost-squid-controller-2/)
```

Make example for this keyboard (after setting up your build environment):

> 104 key default layout:
> 
> ```
> make bpiphany/ghost_squid:default
> ```

Reviewer 2: ```suggestion
    make bpiphany/ghost_squid:default
```

> To directly flash the ghost_squid after compiling use:
> 
> ```
> make bpiphany/ghost_squid:default:dfu
> ```

Reviewer 2: ```suggestion
Flashing example for this keyboard:

    make bpiphany/ghost_squid:default:flash
```
